### DISCUSSION TITLE
Alfa channel doesn't work

### DISCUSSION THREAD

**Discussion by burikus** [SixLabors/ImageSharp.Drawing#201] [Show and tell] [answered]
Hi!
I try to use SixLabors library, but got a problem.
    var color = Color.FromRgba(255, 255, 255, 32);
    var brush = new SolidBrush(color);
Here, no matter which Alfa value is set (0-255) - Always solid color.
**Is it a bug or I do smth wrong ?**

(On the other hand, system.drawing 
var color = Color.FromArgb(32, 255, 255, 255);  works perfect.)

**Comment by JimBobSquarePants** [comment, marked as answer]
You’re using a pixel format that doesn’t contain an alpha component. Use the generic API and Rgba32 or similar

**Reply by burikus** [reply]
Thank you/ I changed to 
`Image.Load<Rgba32>` 
and set 
`Color.FromPixel<Rgba32>(new Rgba32(255, 255, 255, 32));`
now it works.
